.circleci/config.yml

```
@@ -1,20 +1,61 @@
+defaults: &defaults
+  working_directory: ~/repo
+  docker:
+    - image: circleci/node:12.9.1
+
 version: 2
 jobs:
-  build:
-    docker:
-      - image: 5app/node-circleci:2.0.0
-    working_directory: ~/buslane
-    steps: 
+  checkout_code:
+    <<: *defaults
+    steps:
       - checkout
+      - persist_to_workspace:
+          root: ~/repo
+          paths:
+            - .
+
+  test:
+    <<: *defaults
+    steps:
+      - attach_workspace:
+          at: ~/repo
       - restore_cache:
-          key: v2-dependency-cache-{{ checksum "package.json" }}
+          key: dependency-cache-{{ checksum "package.json" }}
       - run:
-          name: Install npm dependencies
+          name: install dependencies with npm
           command: npm install
       - save_cache:
-          key: v2-dependency-cache-{{ checksum "package.json" }}
+          key: dependency-cache-{{ checksum "package.json" }}
           paths:
-            - "node_modules"
+            - ./node_modules
+      - run:
+          name: Lint
+          command: npm run lint
       - run:
-          name: test
-          command: npm test
+          name: Test
+          command: npm run test
```

```suggestion
          command: npm test
```

saves a whole 4 bytes - your welcome